onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -312,7 +342,9 @@ TensorrtExecutionProvider::TensorrtExecutionProvider(const TensorrtExecutionProv
     }
     dump_subgraphs_ = info.dump_subgraphs;
     engine_cache_enable_ = info.engine_cache_enable;
-    if (engine_cache_enable_ || int8_enable_) {
+    timing_cache_enable_ = info.timing_cache_enable;
+    detailed_build_log_ = info.detailed_build_log;
```

Comment:
We also need to add info.force_timing_cache_match here so that the parameter can be passed from provider options.

Reply:
done